client/my-sites/site-settings/navigation.jsx

```diff
@@ -2,37 +2,38 @@
  * External Dependencies
  */
 import React from 'react';
-import i18n from 'i18n-calypso';
-
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
 import config from 'config';
 import SectionNav from 'components/section-nav';
 import NavTabs from 'components/section-nav/tabs';
 import NavItem from 'components/section-nav/item';
+import { getSelectedSite } from 'state/ui/selectors';
 
-export default React.createClass( {
-	displayName: 'SiteSettingsNavigation',
+export class SiteSettingsNavigation extends React.Component {
 
-	propTypes: {
-		site: React.PropTypes.object.isRequired,
+	static propTypes = {
+		site: React.PropTypes.object,
```

Comment:
Can we separate the connected props from the ones we expect to be passed specifically?